lib/magma/loader.rb

```diff
@@ -25,16 +25,6 @@ def entry value
 
   # A generic loader class.
   class Loader
-    class << self
-      def description desc=nil
```

Comment:
Removing this method seems to break the IPI project. I get:

```
magma  | 	15: from /home/developer/magma/lib/magma/server.rb:13:in `initialize'
magma  | 	14: from /home/developer/magma/lib/magma/server.rb:13:in `tap'
magma  | 	13: from /home/developer/magma/lib/magma/server.rb:14:in `block in initialize'
magma  | 	12: from /home/developer/magma/lib/magma.rb:52:in `load_models'
magma  | 	11: from /home/developer/magma/lib/magma.rb:52:in `each'
magma  | 	10: from /home/developer/magma/lib/magma.rb:53:in `block in load_models'
magma  | 	 9: from /home/developer/magma/lib/magma.rb:53:in `new'
magma  | 	 8: from /home/developer/magma/lib/magma/project.rb:29:in `initialize'
magma  | 	 7: from /home/developer/magma/lib/magma/project.rb:84:in `load_project'
magma  | 	 6: from /home/developer/magma/lib/magma/project.rb:92:in `load_project_files'
magma  | 	 5: from /home/developer/magma/lib/magma/project.rb:92:in `require'
magma  | 	 4: from /home/developer/magma/projects/ipi/requirements.rb:18:in `<top (required)>'
magma  | 	 3: from /home/developer/magma/projects/ipi/requirements.rb:18:in `require_relative'
magma  | 	 2: from /home/developer/magma/projects/ipi/loaders/flow_jo.rb:3:in `<top (required)>'
magma  | 	 1: from /home/developer/magma/projects/ipi/loaders/flow_jo.rb:4:in `<module:Ipi>'
magma  | /home/developer/magma/projects/ipi/loaders/flow_jo.rb:5:in `<class:FlowJoLoader>': undefined method `description' for Ipi::FlowJoLoader:Class (NoMethodError)
```

Are you using a branch of the IPI project, or no longer using it? Or is there something I need to tweak locally?